app/model/user/User.java

```diff
@@ -62,4 +65,15 @@ public void setEmailAddress(String emailAddress)
 	{
 		this.emailAddress = emailAddress;
 	}
+	
+	public String getAvatarURL()
+	{
+		return avatarURL;
+	}
+	
+	public String setAvatarURL(String userName)
+	{
+		/* Below URL is of the Crucible ( codereview ) server */
+		this.avatarURL = "http://srv-ind-scrat.vanenburg.com:8060/avatar/" + userName;
```

Comment:
The required size of the avatar (requested with the ?s=number GET parameter) is dependent on the location where it is used. Currently three times 32, and one time 40. 
If you really think it is valuable to make that split right now, maybe  'avatarURLSmall' and avatarURLMedium' or something like that can be introduced for that.